inc/class-core-sitemaps-users.php

```diff
@@ -114,15 +113,18 @@ public function max_num_pages( $type = null ) {
 		// We're not supporting sitemaps for author pages for attachments.
 		unset( $public_post_types['attachment'] );
 
+		if ( null === $page_num ) {
```

Comment:
If we're forcing `paged => 1` here, should that be made the default value of the function?

Reply:
Yeah I had that first, the difference is semantical and paged = 1 makese more sense after reviewing nullable type as a concept, so I will change the code.